.github/workflows/rollout.yml

```
@@ -1,4 +1,5 @@
 name: Rollout to the instances
+run-name: Rollout ${{ inputs.config_tag_name }} to ${{ inputs.config_tag_name }}
```

this would be very useful!
version tag and workflow tag/branch would be different inputs. If that is not easy to extract
```suggestion
run-name: Rollout of Release ${{ inputs.config_tag_name }} 
```